Xwt/Xwt/TreeNavigator.cs

```diff
@@ -115,6 +115,11 @@ public TreeNavigator AddChild ()
 			pos = backend.AddChild (pos);
 			return this;
 		}
+
+		public int GetChildrenCount ()
```

Comment:
This doesn't seem to be related, can we move this into a separate commit pls?

Reply:
Removed. I think it is not needed anymore.